### DISCUSSION TITLE
Jun 28, 2023, uYouPlus is officially broken after development stopped

### DISCUSSION THREAD

**Discussion by WoodpeckerBaby** [qnblackcat/uYouPlus#1169] [General]
https://github.com/arichorn/uYouPlusExtra/issues/40 

YouTube made a server side update today, which broke [uYouPlus](https://github.com/qnblackcat/uYouPlus) v18.14.1-3.0 as in there are lots of ads, blank spaces and videos contain preroll midroll ads as well.

**Comment by duszmox** [comment]
[Here is](https://github.com/qnblackcat/uYouPlus/issues/1160#issuecomment-1599430121) a fix for the ads. It would be good if we wouldn't have to use this workaround, but atleast it is working still.
